pycon/schedule/views.py

```diff
@@ -176,3 +179,55 @@ def slides_upload(request, presentation_id):
     else:
         form = SlidesUploadForm()
         return render(request, "pycon/schedule/slides_upload.html", {'form': form, 'presentation': presentation})
+
+
+@login_required
+def slides_download(request):
+    """Build the slides download page for the captioners."""
+
+    if not request.user.groups.filter(name='captioners').exists():
```

Comment:
`@permission_required('schedule.captioners', raise_exception=True)`

use that decorator in place of `@login_required` above and it will automatically 403 folks, and let superusers in.